### ISSUE TITLE
Navigating to another URL on the FxA form doesn't open about:newtab

### ISSUE THREAD

**Issue by chrismore** [mozilla/all-aboard#277]
Steps to re-produce:

1) Open Firefox

2) Answer onboarding questions

3) Click Go

4) On the FxA form page, go up to the url and type in google.com and hit enter.

Expected results:

A foreground tab opens to about:newtab with the modifications loaded about auto-import

Actual results:

Website changes from first run to whatever URL or search you typed in.

**Comment by chrismore** [comment]
```
// Open a new tab on active window in the background.
tabs.open({
  url: "http://www.mysite.com",
  inBackground: true
});
```

https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/tabs#Open_a_tab

**Comment by FrancescoSTL** [comment]
Closed in #280
